### ISSUE TITLE
Cannot read "found" of undefined

### ISSUE THREAD

**Issue by ninesalt** [spencermountain/compromise#562]
I'm trying to find the conjugation of the verbs in the below sentence but I'm getting an error saying:

> TypeError: Cannot read property 'found' of undefined
>     at getTense (~/project/node_modules/compromise/builds/compromise.js:4559:20)

```
let x = `I can best look after my own daughters, by making sure that my actions seek what is right for all people and all children, and your daughters and your sons.`

console.log(nlp(x).verbs().conjugation())
```

Why is this happening?
PS: Node v8.12, compromise v11.13

**Comment by spencermountain** [comment]
yikes! thanks @ninesalt  - been a long time since we've had a runtime error.
fixed in `11.13.1`

**Comment by ninesalt** [comment]
Do you have an ETA on the release?
